src/kv/RocksDBStore.cc

```diff
@@ -1085,31 +1085,32 @@ int RocksDBStore::get(
     const std::set<string> &keys,
     std::map<string, bufferlist> *out)
 {
+  rocksdb::PinabSlice value;
```

Comment:
```suggestion
  rocksdb:: PinnableSlice value;
```

see 
```
/home/jenkins-build/build/workspace/ceph-pull-requests/src/kv/RocksDBStore.cc: In member function 'virtual int RocksDBStore::get(const string&, const std::set<std::__cxx11::basic_string<char> >&, std::map<std::__cxx11::basic_string<char>, ceph::buffer::v15_2_0::list>*)':
/home/jenkins-build/build/workspace/ceph-pull-requests/src/kv/RocksDBStore.cc:1088:12: error: 'PinabSlice' is not a member of 'rocksdb'; did you mean 'PinnableSlice'?
 1088 |   rocksdb::PinabSlice value;
      |            ^~~~~~~~~~
      |            PinnableSlice
```

also i don't think this change can avoid memcpy, unless `bufferlist` is able to claim the ownership of the buffer possessed by the `PinnableSlice` instance or we can preallocate a Slice/string which can be consumed by `PinnableSlice`.

Reply:
@tchaikov 
Before, rocksdb memtable/blockcache/sstfile copy to string, then append string to buffer. Two copies.

After, value refer to the string in rocksdb memtable/blockcache/sstfile, then append to buffer. one copy

Reply:
@yuyuyu101 thanks for the explanation. thought you meant to say we could save the second copy. will take a closer look at rocksdb to understand how the first copy is eliminated.